tools/binlog/monitor.md

```diff
@@ -0,0 +1,151 @@
+---
+title: TiDB-Binlog Cluster 监控
```

Comment:
Please keep the **title** consistent with the 1st level heading in the body. And also update other ones in this PR.

Reply:
Upated in https://github.com/pingcap/docs-cn/pull/1264/commits/7e117c1135ea6b68cb9bf78d91f45e4cedb3785b